File: QTD.SPA/src/app/_DtoModels/ILA_TaskObjective_Link/ILATaskObjectiveLinkUpdateOptions.ts
import { TaskSequenceModel } from "./TaskSequenceModel";

export class ILATaskObjectiveLinkUpdateOptions {
  taskLinks: TaskSequenceModel[];
  isIncludeProcedures: boolean = false;
  isIncludeEos: boolean = false;
  isIncludeMetaTask: boolean = false;
}
  

File: QTD2.Infrastructure/Model/ILA_TaskObjective_Link/ILATaskObjectiveLinkUpdateOptions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.ILA_TaskObjective_Link
{
    public class ILATaskObjectiveLinkUpdateOptions
    { 
        public List<TaskSequenceModel> TaskLinks { get; set; }
        public bool IsIncludeProcedures { get; set; } = false;
        public bool IsIncludeEos { get; set; } = false;
        public bool IsIncludeMetaTask { get; set; } = false;
    }
}
